samples/local_attestation/common/dispatcher.cpp

```diff
@@ -110,7 +113,7 @@ int ecall_dispatcher::get_target_info(
 exit:
 
     if (report)
-        oe_free_report(report);
+        oe_free_evidence(report);
```

Comment:
If `oe_free_evidence` is preferable, should we also rename report as evidence?

Reply:
Yes. I've gone ahead and changed this.